### PR TITLE
Enable GDAL and fix version.

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@scottwittenburg @aashish24 Please review. We do need gdal in core3d, it turns out.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -43,6 +39,7 @@ requirements:
     - libpng >=1.6.32,<1.6.35
     - jpeg 9*
     - libtiff >=4.0.8,<4.0.10
+    - libgdal
```
Comment (scottwittenburg) [inline recipe/meta.yaml:42]: Is there no need to specify the version required here to match what we depend on everywhere else?  I guess you tried it and it works.

**Comment by scottwittenburg** [comment]
Looks good to me.

File: recipe/meta.yaml (hunk 1 above)
Comment (danlipsa) [reply recipe/meta.yaml:42]: Probably will get in trouble once somebody brings in a gdal that is newer than what we can handle. But we can live with this for now.

Comment (danlipsa) [reply recipe/meta.yaml:42]: I looked again, and seen that pdal does not specify a version either. I guess that's why I did not put one. I think we are fine as long as the spec is the same for pdal and vtk and the gdal that gets used is compatible.

**Pull request merged**